O/OpenVAF/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+using BinaryBuilderBase: get_addable_spec
+
+name = "OpenVAF"
+version = v"23.5.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/pascalkuthe/OpenVAF.git", "d4079e776f4b54b23e158b7857c4e238e5cacd05"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/OpenVAF
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cargo build --release --bin openvaf
+
+install -Dvm 0755 "target/${rust_target}/release/openvaf${exeext}" "${bindir}/openvaf${exeext}"
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+filter!(p -> !(Sys.islinux(p) && libc(p) == "glibc"), platforms)
```

Comment:
Double checking: you did mean this to exclude glibc, and so only build for x86_64-linux-musl?